### DISCUSSION TITLE
Can't apply "&>*" selector in the plugin

### DISCUSSION THREAD

**Discussion by theocerutti** [tailwindlabs/tailwindcss#12998] [Help]
Hello,

I create a button "loading" class and I want to not display any child if this class is applied:

```
  const buttonLoading = {
    position: 'relative',
    '&>*': { // this doesn't work
      display: 'none',
    },
    '&:after': {
      background: theme('backgroundImage.button-loading'),
      ....
    },
  };
``` 

however the selector "&>*" is not applied.
I tried "&>*", "& > *", "&_>_*", "&_*", "&_*", even "*", but nothing works

**Comment by wongjn** [comment]
Seems to work as expected in [this Tailwind Play](https://play.tailwindcss.com/4dZHXY3MFL?file=config) which would suggest some other misconfiguration/misuse in your project.
